packages/nimble-blazor/Examples/Demo.Client/Demo.Client.csproj

```diff
@@ -4,8 +4,13 @@
     <TargetFramework>net6.0</TargetFramework>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+	<RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
```

Comment:
Can you do another pass over the .csproj files to clean up the indentation (tab vs space) issues on these new properties? It looks like all the csproj changes (except for NimbleBlazor.csproj) have tabs on the new parts.